packages/muelu/test/contact/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../unit_tests)
+
+# first ensure that these cmake boolean variables are defined
+ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Tpetra)
+
+IF (${PACKAGE_NAME}_ENABLE_Tpetra)
+
+  TRIBITS_COPY_FILES_TO_BINARY_DIR(Contact
+    SOURCE_FILES
+      segregatedAgg_block_matrix.mm
+      segregatedAgg_filtered_matrix.mm
+      segregatedAgg_matrix.mm
+      segregatedAgg_nullspace.mm
+      segregatedAgg_interface_dof_map_MPI1.mm
+      segregatedAgg_interface_dof_map_MPI3.mm
+  )
+
+  TRIBITS_ADD_EXECUTABLE(
+      SegregatedAggFromFile
+      SOURCES
+      SegregatedAggFromFile.cpp
+      contact_Helpers.cpp
+      COMM serial mpi
+  )
+
+  TRIBITS_ADD_EXECUTABLE(
+      SegregatedAggFromFactory
+      SOURCES
+      SegregatedAggFromFactory.cpp
+      contact_Helpers.cpp
+      COMM serial mpi
+  )
+
+  TRIBITS_ADD_EXECUTABLE(
+      SegregatedAggFromFactoryBlockedA
+      SOURCES
+      SegregatedAggFromFactoryBlockedA.cpp
+      contact_Helpers.cpp
+      COMM serial mpi
+  )
+
+  # The Contact only works with real Scalar types,
+  # so choose an instantiation with real scalars (if any is available)
+  SET(Contact_Instantiation "")
+
+  IF(HAVE_MUELU_INST_DOUBLE_INT_LONGLONGINT)
+    SET(Contact_instantiation "DOUBLE_INT_LONGLONGINT")
+  ELSEIF(HAVE_MUELU_INST_DOUBLE_INT_LONGINT)
+    SET(Contact_instantiation "DOUBLE_INT_LONGINT")
+  ELSEIF(HAVE_MUELU_INST_DOUBLE_INT_INT)
+    SET(Contact_instantiation "DOUBLE_INT_INT")
+  ELSEIF(HAVE_MUELU_INST_FLOAT_INT_INT)
+    SET(Contact_instantiation "FLOAT_INT_INT")
+  ENDIF()
```

Comment:
Let's remove that. Instead, add something like `if (Teuchos::ScalarTraits<Scalar>::isComplex) return;` to the tests.

Reply:
Done. I also removed the same part from the meshtying tests.